Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -82,19 +95,14 @@ spec:
       - ip: 192.168.39.182
 ```
 
-You can use the existing `object-external.yaml` file.
-When ready the ceph-object-controller will output a message in the Operator log similar to this one:
+You can use the existing `object-external.yaml` file. Even though multiple endpoints can be specified, but it is recommend to use only one endpoint.
```

Comment:
here the SC for rgw?

Reply:
Nope in the `object-external.yaml`. Similar to internal multiple RGW instances can point to same set of pools